app-backend/api/src/main/scala/scene/Routes.scala

```diff
@@ -396,4 +403,34 @@ trait SceneRoutes
       }
     }
   }
+
+  def getSentinelMetadata(sceneId: UUID, metadataUrl: String): Route =
+    authenticate { user =>
+      authorizeAsync {
+        val authorizedIO = for {
+          auth <- SceneDao.authorized(user,
+                                      ObjectType.Scene,
+                                      sceneId,
+                                      ActionType.View)
+          datasource <- SceneDao.getSceneDatasource(sceneId)
+        } yield {
+          auth && datasource.id == UUID.fromString(
+            "4a50cb75-815d-4fe5-8bc1-144729ce5b42")
+        }
+        authorizedIO.transact(xa).unsafeToFuture
+      } {
+        onSuccess(
+          SceneDao
+            .getSentinelMetadata(metadataUrl)
+            .transact(xa)
+            .unsafeToFuture) { (s3Object, metaData) =>
+          val s3MediaType = metaData.getContentType() match {
+            case "application/json" => ContentTypes.`application/json`
+            case "application/xml"  => ContentTypes.`text/xml(UTF-8)`
+            case _                  => throw new Exception("Unsupported media type")
```

Comment:
Could complete with a 415 here instead of throwing an exception, which will come back as a 503